app/src/main/java/com/jerboa/db/AppDB.kt

```diff
@@ -325,8 +330,17 @@ val MIGRATION_10_11 = object : Migration(10, 11) {
     }
 }
 
+val MIGRATION_11_12 = object : Migration(11, 12) {
```

Comment:
Actually, can you make this part of the 10_11 migration? Let's keep all the settings migrations there until 0.0.33 is released.

Reply:
Good point! This confused me as I must have had V11 in my local build, then made changes, and got a schema exception on startup. But I think I just needed to clear my app data... should be fixed now, I've moved it into V11 with the other settings.